src/runtime/virtcontainers/mount_test.go

```diff
@@ -205,22 +205,24 @@ func TestGetDeviceForPathBindMount(t *testing.T) {
 	assert.Equal(sourceDev, destDev)
 }
 
-func TestIsDeviceMapper(t *testing.T) {
+func TestCheckBlockDevice(t *testing.T) {
 	assert := assert.New(t)
 
 	// known major, minor for /dev/tty
 	major := 5
 	minor := 0
 
-	isDM, err := isDeviceMapper(major, minor)
+	isBD, err := checkBlockDevice(major, minor)
 	assert.NoError(err)
-	assert.False(isDM)
+	assert.False(isBD)
 
 	// fake the block device format
-	blockFormatTemplate = "/sys/dev/char/%d:%d"
-	isDM, err = isDeviceMapper(major, minor)
+	blockFormatTemplates = map[string]string{
```

Comment:
Better to reset `blockFormatTemplates` so in case other case will not be impacted, I think the old code probably missed this part

```
	blockFormatTemplatesOld := blockFormatTemplates
	defer func() {
		blockFormatTemplates = blockFormatTemplatesOld
	}()
	blockFormatTemplates = map[string]string{
		"/dev/tty": "/sys/dev/char/%d:%d",
	}
```

Reply:
It's a good way, I have done so.